### PR TITLE
Player to userid

### PR DESCRIPTION
**Pull request by yuriyoleynikov**

### CODE REVIEW

File: src/YoTennis/Controllers/MatchController.cs [hunk 1]
```diff
@@ -48,16 +49,54 @@ public MatchController(UserManager<ApplicationUser> userManager, IMatchListServi
...
+        {
+            var matchService = await _matchListService.GetMatchService(UserId, id);
+
+            var e = new ChangePlayersEvent()
+            {
+                OccuredAt = DateTime.UtcNow
+            };
+
+            if (player == Player.First)
+                e.FirstPlayerUserId = "none";
```
Comment (Igorbek) [inline src/YoTennis/Controllers/MatchController.cs]: use `null`

File: src/YoTennis/Models/MatchDetailsViewModel.cs [hunk 2]
```diff
@@ -9,5 +9,6 @@ public class MatchDetailsViewModel
     {
         public string Id { get; set; }
         public MatchModel MatchModel { get; set; }
+        public string UserId { get; set; }
```
Comment (Igorbek) [inline src/YoTennis/Models/MatchDetailsViewModel.cs:12]: do not expose user id, just use user name

File: src/YoTennis/Services/DatabaseMatchService.cs [hunk 3]
```diff
@@ -120,5 +120,12 @@ public Task Reset()
 
             return _gameHandler.PlayersStats;
         }
+
+        public async Task PlayerToUserAsync(Player player)
```
Comment (Igorbek) [inline src/YoTennis/Services/DatabaseMatchService.cs]: what does this method do?

**Review by Igorbek — CHANGES_REQUESTED** [review]

File: tests/YoTennis.Tests/Test/HandlerSnapshotTests.cs [hunk 4]
```diff
@@ -11,7 +11,7 @@ namespace YoTennis.Tests.Test
     using YoTennis.Tests.Helpers;
 
     public class HandlerSnapshotTests
-    {
+    {/*
```
Comment (Igorbek) [inline tests/YoTennis.Tests/Test/HandlerSnapshotTests.cs]: why did you comment the tests?

**Comment by Igorbek** [comment]
Are you going to complete/merge this?

File: src/YoTennis/Controllers/MatchController.cs (hunk 1 above)
Comment (yuriyoleynikov) [reply src/YoTennis/Controllers/MatchController.cs]: ok

**Review by Igorbek — APPROVED** [review]

**Pull request merged**